packages/plugins/rubygem-foreman_snapshot_management/rubygem-foreman_snapshot_management.spec

```diff
@@ -94,8 +78,6 @@ cp -pa .%{gem_dir}/* \
 %exclude %{gem_cache}
 %{gem_spec}
 %{foreman_bundlerd_plugin}
-%{foreman_assets_plugin}
-%{foreman_assets_foreman}
```

Comment:
Those two lines need to remain, it's a bug in our template :/

https://github.com/theforeman/foreman-packaging/pull/9442